roundabout/search/views.py

```diff
@@ -297,9 +297,10 @@ class InventoryTableView(GenericSearchTableView):
     def get_avail_fields():
         avail_fields = [dict(value="part__name",              text="Name", legal_lookups='STR_LOOKUP'),
                         dict(value="serial_number",  text="Serial Number", legal_lookups='STR_LOOKUP'),
+                        dict(value="old_serial_number", text="Old Serial Number", legal_lookups='STR_LOOKUP'),
                         dict(value="location__name",      text="Location", legal_lookups='STR_LOOKUP'),
                         dict(value="build__assembly__name",  text="Build", legal_lookups='STR_LOOKUP'),
-                        dict(value="revision__note",          text="Note", legal_lookups='STR_LOOKUP'),
+                        dict(value="detail",                text="Detail", legal_lookups='STR_LOOKUP'),
```

Comment:
@sbatchelder Can we call this "Note" still, instead of "Detail"? I think using "Note" will be more clear to users regarding what they are searching on, even though it may be a bit confusing to us as developers. If anyone (@cahearn ) has different thoughts on this, please speak up... I'm _not_ actually an end user, even though I often decide what's "best" for them!

Reply:
Tough one.  Using "Detail" means users will have to figure out/be told to use this option to search for Notes.  Using "Note" will be more clear for the expected most common use but will be confusing if the search is on anything in the details and returns other results.  Also could lead to limited use as users may assume search is limited to Note details, not realizing they can search details of all actions.  "Detail" has the bonus of being clear for developers.  Since both will require clarification I lean toward keeping it "Detail" if/until we choose to refine the search and limit to Detail text of Note actions.
Curveball would be "Details/Note" or "Note/Details," which is immediately calming these nerves...

Reply:
Good idea... I vote "Notes & Details" for the best clarity (and the inconsistent pluralization was bothering me). :) @sbatchelder this is ready for update.